2/contrib/openshift/configuration/config.xml.tpl

```diff
@@ -44,13 +44,14 @@
   <views>
     <hudson.model.AllView>
       <owner class="hudson" reference="../../.."/>
-      <name>all</name>
+      <name>All</name>
```

Comment:
just undoing that change to have it the same as upstream again

Reply:
jenkins will change that back to lowercase on startup anyway, should probably be changed in upstream

Reply:
but this way if we diff against upstream we don't see this change and have to wonder why this changed

Reply:
true that.  Maybe™ we'll create an upstream PR for this at one point.